package.json

```diff
@@ -65,10 +65,10 @@
     "visual-testing-app:serve": "serve visual-testing-app/dist --no-clipboard --single --listen 3001"
   },
   "dependencies": {
-    "@emotion/core": "10.0.16",
-    "@emotion/styled": "10.0.15",
+    "@emotion/core": "10.0.17",
+    "@emotion/styled": "10.0.17",
     "common-tags": "1.8.0",
-    "dom-helpers": "3.4.0",
+    "dom-helpers": "5.0.1",
```

Comment:
Oh wow. What changed here?

Reply:
https://github.com/react-bootstrap/dom-helpers/commit/87e2fe448b5e852574e1ee63bd3f2102b4c65d79

Looks like they dropped IE 8 support and "removed a lot of code"